bitrixSM/internal/services/command/machine.go

```diff
@@ -5,7 +5,8 @@ import (
 	"log/slog"
 )
 
-func AddShiftWorker(webhookURL string, args []string, log *slog.Logger) (string, error) {
+func AddMachine(baseURL string, args []string, log *slog.Logger) (string, error) {
+	webhookURL := baseURL + "/machine"
```

Comment:
use structure to keep all routing - related details. Don'w use magic string - it will be very very hard (almost impossible) to refactor system routing without single point of truth